homeassistant/components/deconz/logbook.py

```diff
@@ -148,6 +149,7 @@ def async_describe_deconz_event(event: Event) -> dict:
         deconz_event: DeconzEvent | None = _get_deconz_event_from_device_id(
             hass, event.data[ATTR_DEVICE_ID]
         )
+        assert deconz_event
```

Comment:
Why is this assert needed? Avoid the use of asserts.

Reply:
Fixed